From 4702a0cde506774bc07375b013d03741c1b089f0 Mon Sep 17 00:00:00 2001
From: Rob Browning <rlb@defaultvalue.org>
Date: Fri, 12 Aug 2016 20:21:43 -0500
Subject: Look for guile-procedures.txt in pkglibdir

Look for guile-procedures.txt in %guile-build-info's pkglibdir first
since the content is not architecture independent.

Bug: https://debbugs.gnu.org/22833
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=816123
---
 module/ice-9/documentation.scm | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/module/ice-9/documentation.scm b/module/ice-9/documentation.scm
index 9b0a121cc..a0d533ee6 100644
--- a/module/ice-9/documentation.scm
+++ b/module/ice-9/documentation.scm
@@ -150,7 +150,10 @@
 (define documentation-files
   (map (lambda (vicinity)
 	 (in-vicinity (vicinity) "guile-procedures.txt"))
-       (list %library-dir
+       (list (lambda ()
+               (string-append (cdr (assq 'pkglibdir %guile-build-info))
+                              "/" (effective-version)))
+             %library-dir
 	     %package-data-dir
 	     %site-dir
 	     (lambda () "."))))
